### PR TITLE
[Chore] Publish with github packages

### PR DESCRIPTION
**Pull request by attilakiss323**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        id: autotagger
+        uses: butlerlogic/action-autotag@stable
+        with:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          tag_prefix: "v"
+
+      - name: Release
+        id: create_release
+        if: steps.autotagger.outputs.tagname != ''
+        uses: actions/create-release@v1.1.4
```
Comment (sebastian-schlecht) [inline .github/workflows/main.yml:25]: is this factually creating a release?

Comment (sebastian-schlecht) [reply .github/workflows/main.yml:25]: as in https://github.com/capmo/monitoring/blob/main/.github/workflows/release-package.yml ?

Comment (attilakiss323) [reply .github/workflows/main.yml:25]: I think so. It says `This new workflow will run automatically every time you push a change to a branch named release in the repository. You can view the progress in the Actions tab.` here https://docs.github.com/en/packages/guides/using-github-packages-with-github-actions

**Review by kotielnikov — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -42,3 +45,30 @@ npm install eslint-plugin-import@^2.22.0 \
...
+
+## Setup
+
+1. Install @capmo/config package
+2. Create a dangerfile in the root of the repo with the following content:
+
+```
+const { dangerJs } = require('@capmo/config/Danger/dangerfile');
+
+dangerJs();
```
Comment (across-the-grid) [inline README.md:58]:
```suggestion
dangerJs(); // Optional: set a specific platform
```

**Review by across-the-grid — APPROVED** [review]
Just saw this PR, thanks for addressing my comment! 👍

File: README.md (hunk 2 above)
Comment (attilakiss323) [reply README.md:58]: Its described below. I don't want people copy-pasting `// Optional: set a specific platform` in repos

**Review by kotielnikov — APPROVED** [review]

**Pull request merged**